Migrate account_compute_tax_amount to 8.0

guewen opened the pull request:
- New API
- A better description of the module
- Do no longer hides the field but set it readonly

Commit:
More detailed description for account_compute_tax_amount, the current
one is lacunar

Commit:
Move account_compute_tax_amount from __unported__ to root

Commit:
Set account_compute_tax_amount to installable

Commit:
Migrate to new API

Commit:
'invisible' is not propagated from the model to the view

Anyway, set the field readonly is better than invisible because
it is still interesting to see this value even if it cannot be changed

Commit:
update copyrights

pedrobaeza commented on account_compute_tax_amount/README.rst:
```diff
@@ -0,0 +1,13 @@
+The way the tax amount is computed in the core `account` module is prone
+to input errors.
+
+This module forces the the tax amount to always be: `credit - debit`
```
double `the`

pedrobaeza commented on account_compute_tax_amount/README.rst:
```diff
@@ -0,0 +1,13 @@
+Recompute tax amount
+====================
+
+The way the tax amount is computed in the core `account` module is prone
```
You should put `in account journal entries`

Commit:
Improve the readme: more context and correct the inline literals with
double backticks

Commit:
Fix signature compatibility issues for create() and write()

On account.move.line, create() and write() have additional arguments so
we have to use the @cr_uid... decorators.

Commit:
too many blank lines

nbessi commented:
:+1:

pedrobaeza commented:
:+1:
